src/xGadgetron/cGadgetron/tests/mrtests.cpp

```diff
@@ -641,12 +642,15 @@ int main ( int argc, char* argv[])
 		
         std::string SIRF_PATH;
         if (argc==1)
-            SIRF_PATH = sirf::getenv("SIRF_PATH", true);
+        {
+            std::stringstream sssirfpath;
+            sssirfpath << sirf::getenv("SIRF_INSTALL_PATH", true) << "/share/SIRF-" << SIRF_VERSION_MAJOR << "." << SIRF_VERSION_MINOR;
+            SIRF_PATH = sssirfpath.str();
+        }
         else
             SIRF_PATH = argv[1];
 
-//        std::string data_path = SIRF_PATH + "/data/examples/MR/simulated_MR_2D_cartesian_Grappa2.h5";
-        std::string data_path = SIRF_PATH + "/data/examples/MR/simulated_MR_2D_cartesian.h5";
+        std::string data_path = SIRF_PATH + "/data/examples/MR/autogenerated_simulated_MR_2D_cartesian.h5";
```

Comment:
does this path concatenation work on Windows?

Reply:
it wouldn't